packages/compose/index.ts

```diff
@@ -177,7 +177,7 @@ const createFactory: CreateFactory = () => {
             stamp: Stamp as Stamp,
             args,
           });
-          if (returnedValue !== undefined) instance = returnedValue;
+          if (returnedValue !== undefined) instance = returnedValue as object;
```

Comment:
Returned values are not always objects. The specs say that it can be anything except `undefined`. Meaning: object, string, boolean, array, number, you name it.
Maybe `any` or `unknown` is better here?

Reply:
I was under the impression that it had to be object. Will try unknown first.

Reply:
See comment in next review for explanation.